modular_chomp/code/modules/mob/living/simple_mob/subtypes/animal/tyr/tyr_base.dm

```diff
@@ -0,0 +1,44 @@
+#define FACTION_TYR "tyr"
+
+/mob/living/simple_mob/animal/tyr
+	minbodytemp = 175
+	maxbodytemp = 500
+	cold_resist = -0.5
+	heat_resist = 0.75
+
+	see_in_dark = 3 //stealth time?
+
+	icon = 'modular_chomp/icons/mob/tyr.dmi'
+
+	faction = "tyr"
```

Comment:
factions use defines now

Reply:
I was dumb when reading the define thing initially, sorry.